src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -73,15 +75,23 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
 
         private static TokenValidationParameters CreateTokenValidationParameters()
         {
+            List<SymmetricSecurityKey> signingKeys = new List<SymmetricSecurityKey>();
+            string defaultKey = Util.GetDefaultKeyValue();
```

Comment:
This line is what we had before - previously this was the ONLY key used.

Reply:
It's a bit confusing to have this in multiple places, as opposed to all consolidated in the `TryGetEncryptionKey` method. Are you concerned about the default probing done by our Data Protection layer?

Reply:
The issue is that SecretsUtility.TryGetEncryptionKey has an existing key ordering that is used for encryption/decryption of data, outside of JWT token use. It's also used by SimpleWebTokenHelper where the TryGetEncryptionKey logic previously existed before being refactored out into SecretsUtility for reuse in https://github.com/Azure/azure-functions-host/commit/80f5ebcaba334a79ab049e2fe276dc15f257e554. For the usages in SimpleWebTokenHelper, changing the ordering would result in breaking changes on that side.

Reply:
I've factored this out into a helper with some explanatory comments